src/code/game.c

```diff
@@ -54,28 +54,26 @@ void GameState_SetFBFilter(Gfx** gfx, u32 arg1) {
                 sMonoColors.envColor.g = R_FB_FILTER_ENV_COLOR(1);
                 sMonoColors.envColor.b = R_FB_FILTER_ENV_COLOR(2);
                 sMonoColors.envColor.a = R_FB_FILTER_A;
-                VisMono_Draw(&sMonoColors, &_gfx, arg1);
+                VisMono_Draw(&sMonoColors, &gfxP, arg1);
             }
         }
     }
 
-    *gfx = _gfx;
+    *gfx = gfxP;
 }
 
-void Game_Nop80173534(GameState* gamestate) {
+void Game_Nop80173534(GameState* gameState) {
     ;
 }
```

Comment:
Is the semicolon needed?
```suggestion
void Game_Nop80173534(GameState* gameState) {
}
```